CHANGELOG.md

```diff
@@ -1,3 +1,115 @@
+(configurable) number of `npm-debug.log` files, which you can access in the
+future. Hopefully this will help clean stuff up and reduce frustration from
+missed repros! In the future, this will also be used by `npm report` to make it
+super easy to put up issues about crashes you run into with npm. 💃🕺🏿👯‍♂️
+
+* [`04fca22`](https://github.com/npm/npm/commit/04fca223a0f704b69340c5f81b26907238fad878)
+  [#11439](https://github.com/npm/npm/pull/11439)
+  Put debug logs in `$(npm get cache)/_logs` and store multiple log files.
+  ([@KenanY](https://github.com/KenanY))
+  ([@othiym23](https://github.com/othiym23))
+  ([@isaacs](https://github.com/isaacs))
+  ([@iarna](https://github.com/iarna))
+
+#### DOCS
+
+* [`ae8e71c`](https://github.com/npm/npm/commit/ae8e71c2b7d64d782af287a21e146d7cea6e5273)
+  [#15402](https://github.com/npm/npm/pull/15402)
+  Add missing backtick in one of the `npm doctor` messages.
+  ([@watilde](https://github.com/watilde), [@charlotteis](https://github.com/charlotteis))
+* [`821fee6`](https://github.com/npm/npm/commit/821fee6d0b12a324e035c397ae73904db97d07d2)
+  [#15480](https://github.com/npm/npm/pull/15480)
+  Clarify that unscoped packages can depend on scoped packages and vice-versa.
+  ([@chocolateboy](https://github.com/chocolateboy))
+* [`2ee45a8`](https://github.com/npm/npm/commit/2ee45a884137ae0706b7c741c671fef2cb3bac96)
+  [#15515](https://github.com/npm/npm/pull/15515)
+  Update minimum supported Node version number in the README to `node@>=4`.
+  ([@watilde](https://github.com/watilde))
+* [`af06aa9`](https://github.com/npm/npm/commit/af06aa9a357578a8fd58c575f3dbe55bc65fc376)
+  [#15520](https://github.com/npm/npm/pull/15520)
+  Add section to `npm-scope` docs to explain that scope owners will own scoped
+  packages with that scope. That is, user `@alice` is not allowed to publish to
+  `@bob/my-package` unless explicitly made an owner by user (or org) `@bob`.
+  ([@hzoo](https://github.com/hzoo))
+* [`bc892e6`](https://github.com/npm/npm/commit/bc892e6d07a4c6646480703641a4d71129c38b6d)
+  [#15539](https://github.com/npm/npm/pull/15539)
+  Replace `http` with `https` and fix typos in some docs.
+  ([@watilde](https://github.com/watilde))
+* [`1dfe875`](https://github.com/npm/npm/commit/1dfe875b9ac61a0ab9f61a2eab02bacf6cce583c)
+  [#15545](https://github.com/npm/npm/pull/15545)
+  Update Node.js download link to point to the right place.
+  ([@watilde](https://github.com/watilde))
+
+#### MISC
+
+* [`98df212`](https://github.com/npm/npm/commit/98df212a91fd6ff4a02b9cd247f4166f93d3977a)
+  [#15492](https://github.com/npm/npm/pull/15492)
+  Update the "master" node version used for AppVeyor to `node@7`.
+  ([@watilde](https://github.com/watilde))
+* [`d75fc03`](https://github.com/npm/npm/commit/d75fc03eda5364f12ac266fa4f66e31c2e44e864)
+  [#15413](https://github.com/npm/npm/pull/15413)
+  `npm run-script` now exits with the child process' exit code on exit.
+  ([@kapals](https://github.com/kapals))
+
```

Comment:
I notice there are no dependency updates here? It looks like `ansi-regex` and `mississippi` are out of date.